2025-08-12 15:13:54,748 INFO    MainThread:6088 [wandb_setup.py:_flush():68] Current SDK version is 0.19.10
2025-08-12 15:13:54,748 INFO    MainThread:6088 [wandb_setup.py:_flush():68] Configure stats pid to 6088
2025-08-12 15:13:54,748 INFO    MainThread:6088 [wandb_setup.py:_flush():68] Loading settings from /home/fdittaro/.config/wandb/settings
2025-08-12 15:13:54,748 INFO    MainThread:6088 [wandb_setup.py:_flush():68] Loading settings from /media/HDD/fdittaro/Third_training_phase/CropAndWeed/wandb/settings
2025-08-12 15:13:54,748 INFO    MainThread:6088 [wandb_setup.py:_flush():68] Loading settings from environment variables
2025-08-12 15:13:54,748 INFO    MainThread:6088 [wandb_init.py:setup_run_log_directory():724] Logging user logs to /media/HDD/fdittaro/Third_training_phase/CropAndWeed/wandb/run-20250812_151354-zrnvfqw7/logs/debug.log
2025-08-12 15:13:54,748 INFO    MainThread:6088 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to /media/HDD/fdittaro/Third_training_phase/CropAndWeed/wandb/run-20250812_151354-zrnvfqw7/logs/debug-internal.log
2025-08-12 15:13:54,748 INFO    MainThread:6088 [wandb_init.py:monkeypatch_ipython():643] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f39ddfe86d0>
2025-08-12 15:13:54,749 INFO    MainThread:6088 [wandb_init.py:init():852] calling init triggers
2025-08-12 15:13:54,749 INFO    MainThread:6088 [wandb_init.py:init():857] wandb.init called with sweep_config: {}
config: {'epochs': 50, 'batch_size': 32, '_wandb': {}}
2025-08-12 15:13:54,749 INFO    MainThread:6088 [wandb_init.py:init():893] starting backend
2025-08-12 15:13:54,749 INFO    MainThread:6088 [wandb_init.py:init():897] sending inform_init request
2025-08-12 15:13:54,756 INFO    MainThread:6088 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-08-12 15:13:54,756 INFO    MainThread:6088 [wandb_init.py:init():907] backend started and connected
2025-08-12 15:13:54,761 INFO    MainThread:6088 [wandb_run.py:_label_probe_notebook():1369] probe notebook
2025-08-12 15:13:54,761 INFO    MainThread:6088 [wandb_run.py:_label_probe_notebook():1379] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-08-12 15:13:54,761 INFO    MainThread:6088 [wandb_init.py:init():1002] updated telemetry
2025-08-12 15:13:54,762 INFO    MainThread:6088 [wandb_init.py:init():1026] communicating run to backend with 90.0 second timeout
2025-08-12 15:13:55,369 INFO    MainThread:6088 [wandb_init.py:init():1101] starting run threads in backend
2025-08-12 15:13:55,496 INFO    MainThread:6088 [wandb_run.py:_console_start():2566] atexit reg
2025-08-12 15:13:55,496 INFO    MainThread:6088 [wandb_run.py:_redirect():2414] redirect: wrap_raw
2025-08-12 15:13:55,496 INFO    MainThread:6088 [wandb_run.py:_redirect():2483] Wrapping output streams.
2025-08-12 15:13:55,496 INFO    MainThread:6088 [wandb_run.py:_redirect():2506] Redirects installed.
2025-08-12 15:13:55,498 INFO    MainThread:6088 [wandb_init.py:init():1147] run started, returning control to user process
2025-08-12 15:13:55,499 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:15:23,695 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:15:23,695 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:15:23,842 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:15:36,199 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:15:36,199 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:15:36,206 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:15:42,507 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:15:42,508 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:15:42,510 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:15:47,296 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:15:47,296 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:15:47,381 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:18:14,479 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:18:14,480 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:18:14,481 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:18:26,912 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:18:26,912 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:18:26,913 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:19:55,024 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:19:55,024 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:19:55,880 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:20:06,584 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:20:06,584 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:20:06,585 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:20:10,645 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:20:10,645 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:20:10,646 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:20:16,667 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:20:16,668 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:20:25,400 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:20:35,264 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:20:35,264 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:20:35,265 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:20:42,792 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:20:42,793 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:20:47,488 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:21:25,733 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:21:25,734 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:21:25,735 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:21:39,266 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:21:39,267 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:21:39,271 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:21:41,773 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:21:41,773 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:21:41,774 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:21:43,221 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:21:43,221 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 15:21:43,223 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 15:21:45,143 INFO    MainThread:6088 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-08-12 15:21:45,143 INFO    MainThread:6088 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-08-12 16:18:45,459 INFO    MainThread:6088 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-08-12 16:19:09,557 INFO    MsgRouterThr:6088 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
2025-08-12 16:19:11,360 ERROR   Thread-2  :6088 [redirect.py:_on_write():664] [no run ID] error in stdout callback
Traceback (most recent call last):
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 662, in _on_write
    cb(written_data)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 2487, in <lambda>
    lambda data: self._console_raw_callback("stdout", data),
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 464, in wrapper_fn
    return func(self, *args, **kwargs)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1632, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 763, in publish_output_raw
    self._publish_output_raw(o)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/interface/interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/lib/sock_client.py", line 174, in send_record_publish
    self.send_server_request(server_req)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/lib/sock_client.py", line 154, in send_server_request
    self._send_message(msg)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/lib/sock_client.py", line 151, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
